test/e2e/set/scyllacluster/scyllacluster_scaling.go

```diff
@@ -173,12 +186,17 @@ var _ = g.Describe("ScyllaCluster", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		verifyScyllaCluster(ctx, f.KubeClient(), sc)
+		waitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
 
 		oldHosts = hosts
-		hosts = getScyllaHostsAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
+		oldHostIDs = hostIDs
+		hosts, hostIDs, err = utils.GetBroadcastRPCAddressesAndUUIDs(ctx, f.KubeClient().CoreV1(), sc)
+		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(oldHosts).To(o.HaveLen(2))
+		o.Expect(oldHostIDs).To(o.HaveLen(2))
 		o.Expect(hosts).To(o.HaveLen(1))
-		o.Expect(oldHosts).To(o.ContainElements(hosts))
+		o.Expect(hostIDs).To(o.HaveLen(1))
+		o.Expect(oldHostIDs).To(o.ContainElements(hostIDs))
```

Comment:
shouldn't this be `ConsistOf` so it implies the sets should be equal in random order? I think `ContainElements` means that the `oldHostIDs` could have extra items.
(globally)

Reply:
They won't be equal - the cluster was just scaled down (2->1).